pkg/clusteragent/admission/metrics/metrics.go

```diff
@@ -16,13 +16,8 @@ import (
 
 // Metric names
 const (
-	SecretControllerName     = "secrets"
-	WebhooksControllerName   = "webhooks"
-	TagsMutationType         = "standard_tags"
-	ConfigMutationType       = "agent_config"
-	LibInjectionMutationType = "lib_injection"
-	CWSPodInstrumentation    = "cws_pod_instrumentation"
-	CWSExecInstrumentation   = "cws_exec_instrumentation"
```

Comment:
These are not needed since we are using the same value for the webhook name, so we can set the tag value directly using the webhook name